packages/client/src/generation/utils/dedent.ts

```diff
@@ -0,0 +1,47 @@
+// https://github.com/dmnd/dedent/blob/master/dedent.js
```

Comment:
Is it best to add this in package json? I did not do it because I was unsure about the dev/prod dependencies.

Reply:
add it to package.json as a dev dependency, it will get bundled